core/domain/feedback_services.py

```diff
@@ -139,81 +139,142 @@ def create_message(
         received_via_email: bool. Whether new message is received via email or
             web.
     """
+    create_messages(
+        [thread_id], author_id, updated_status, updated_subject, text,
+        received_via_email=received_via_email)
 
+
+def create_messages(
+        thread_ids, author_id, updated_status, updated_subject, text,
+        received_via_email=False):
+    """Creates a new message for each of the threads in thread_ids and
+    for each message, subscribes the author to the thread.
+
+    Args:
+        thread_ids: list(str). The thread ids the messages belong to.
```

Comment:
Perhaps add a note that all these should be distinct, and verify at the outset of the function's execution that they all are (otherwise, throw an error).